packages/api/lib/granule-delete.js

```diff
@@ -64,7 +64,7 @@ const deleteGranuleAndFiles = async ({
         }
       );
 
-      await granulePgModel.delete(trx, { granule_id: pgGranule.granule_id });
+      await granulePgModel.delete(trx, { cumulus_id: pgGranule.cumulus_id });
```

Comment:
Already updated in https://github.com/nasa/cumulus/pull/2091

Made the change here to test changes.